ta/pkcs11/include/pkcs11_ta.h

```diff
@@ -1188,6 +1188,7 @@ enum pkcs11_attr_id {
 						  0x0600,
 
 	/* Vendor specific attributes */
+	PKCS11_CKA_INDESTRUCTIBLE = 0x80000010,
```

Comment:
```suggestion
	PKCS11_CKA_INDESTRUCTIBLE = PKCS11_CKA_VENDOR_DEFINED |
					  0x0001,
```
And move below `PKCS11_CKA_OPTEE_HIDDEN_EC_POINT` definition.